src/tools/web/clusterd/src/db.js

```diff
@@ -0,0 +1,44 @@
+import low from 'lowdb'
```

Comment:
I thought it would be easier to use a simple JSON based db here, but it turns out it's not. I would have to wrap the elektra configuration into a cluster configuration (in lowdb) and unwrap it again later, and then special cases aren't handled yet (like metadata, etc).

I'll have to refactor clusterd to use elektra for its database, then I can simply set the keys as I would with a normal instance and push the changes to the instances assigned to the cluster.